.gitignore

```diff
@@ -18,6 +18,7 @@ comparer-config.toml
 #ignore cmake cache
 /build-*/
 .vscode/tasks.json
+/.cache/
```

Comment:
Why?

Reply:
The folder was showing up to git with a few hundred files when I did a build (linux).  Entry doesn't seem to be working though

Reply:
Should rather remove those changes rather than using gitignore.

Reply:
Removed